modules/generate-network-policies-ng20.adoc

```diff
@@ -8,14 +8,26 @@
 
 .Procedure
 . In the {product-title-short} portal, navigate to *Network Graph*.
 . Select a cluster, and then select one or more namespaces.
-. In the network graph header, select *Simulate network policy*. {product-title-short} generates policies for all of the deployments that exist in the selected cluster.
+. Optional: Select individual deployments to restrict the policy generated to just those deployments. You can also use the *Filter deployments* feature to further narrow the scope.
+. In the network graph header, select *Network policy generator*.
 . Optional: In the information panel that opens, select *Exclude ports & protocols* if you do not want ports and protocols to be scoped in {product-title-short} generated policies.
-. Select *Generate and simulate network policies*.
-The generated network policy configuration YAML file opens in the same panel, and the network graph shows the effects of the policies.
+. Click *Generate and simulate network policies*. {product-title-short} generates policies for the scope that you have chosen. This scope is displayed at the top of the *Generate network policies* panel. Clicking on the deployment information in the scope displays a list of the deployments that are included.
+. The generated network policy configuration YAML file opens in the same panel, and the network graph shows the effects of the policies. You can copy the displayed YAML file to the clipboard or download it by clicking the appropriate icon in the panel.
+. Optional: To compare the generated network policies to the existing network policies, click *Compare*. The YAML files for existing and generated network policies are shown in a side-by-side view.
++
+[NOTE]
+====
+Some items do not have generated policies, such as namespaces with existing ingress policies or deployments in certain protected namespaces such as as `stackrox` or `acs`.
+====
+. Optional: Click the *Actions* menu to perform the following activities:
+* Share the YAML file with notifiers: Sends the YAML file to one of the system notifiers you have configured (for example, Slack, ServiceNow, or an application that uses generic webhooks). These notifiers are configured by navigating to *Platform Configuration* -> *Integrations*. See the documentation in the "Additional resources" section for more information.
+* Rebuild rules from active traffic: Gathers a new baseline for the network policy and rebuilds the network policy from the baseline. ##Is this correct?##
+* Revert rules to previously applied YAML: Removes the simulated policy and reverts to the last network policy.
```

Comment:
@dvail are you referring to the reverting? This behavior should be exactly the same as what we had before. It's basically the undo functionality from the old network graph. If we had applied a new network policy A->B. Then it reverts back to A

Reply:
@sachaudh How do you apply a policy to the current network graph? I thought the ability to apply automatically generated policies was removed - or are you talking about different functionality? In the flows I am testing the "Revert rules" button always results in an error in the UI, since there is no action applied to the current cluster to undo.

Reply:
it should be the same as before like saif was mentioning afaik. i think you might be seeing errors @dvail since there is no policy to revert to. idt the way the policy was generated or added matters for this functionality (can be uploaded or generated) but i think it can only be reverted if there was a policy previously applied.